site/src/components/WorkspaceScheduleBanner/WorkspaceScheduleBanner.tsx

```diff
@@ -1,20 +1,28 @@
+import Button from "@material-ui/core/Button"
 import Alert from "@material-ui/lab/Alert"
 import AlertTitle from "@material-ui/lab/AlertTitle"
+import { useMachine } from "@xstate/react"
 import dayjs from "dayjs"
 import isSameOrBefore from "dayjs/plugin/isSameOrBefore"
 import utc from "dayjs/plugin/utc"
 import { FC } from "react"
 import * as TypesGen from "../../api/typesGenerated"
 import { isWorkspaceOn } from "../../util/workspace"
+import { workspaceScheduleBanner } from "../../xServices/workspaceSchedule/workspaceScheduleBannerXService"
 
 dayjs.extend(utc)
 dayjs.extend(isSameOrBefore)
 
 export const Language = {
+  bannerAction: "Extend",
   bannerTitle: "Your workspace is scheduled to automatically shut down soon.",
 }
 
 export interface WorkspaceScheduleBannerProps {
+  /**
+   * @remarks __onExtend is used for testing purposes
+   */
+  __onExtend?: () => void
```

Comment:
Is there because there's not a convenient way to mock `bannerSend`? Why does that matter in a story?

Reply:
it matters because `bannerSend` hits the API

Reply:
But only `onClick`, right? I guess we'd be concerned about the canvas snapshot? 
I'm all for safe rather than story - just trying to get a better understanding of storybook.